Sources/Extensions/CALayer+Extensions.swift

```diff
@@ -125,7 +125,11 @@ extension CALayer {
     private func calculateNumLines(for config: SkeletonMultilinesLayerConfig) -> Int {
         let definedNumberOfLines = config.lines
         let requiredSpaceForEachLine = config.lineHeight + config.multilineSpacing
-        let calculatedNumberOfLines = Int(round(CGFloat(bounds.height - config.paddingInsets.top - config.paddingInsets.bottom) / CGFloat(requiredSpaceForEachLine)))
+
+        let neededLines = round(CGFloat(bounds.height - config.paddingInsets.top - config.paddingInsets.bottom) / CGFloat(requiredSpaceForEachLine))
+        guard !neededLines.isNaN else { return 0 }
```

Comment:
taking into account the crash description we should check if the value is finite as well. In fact, I think we could use the property `isNormal`. As we can see in the [official doc](https://developer.apple.com/documentation/coregraphics/cgfloat/1456300-isnormal), this property return `false` is the value is `zero`, `infinite` or `NaN`

I suggest you modify the `guard` condition:
```suggestion
        guard neededLines.isNormal else { return 0 }
```

WDYT?